docker/conf/cargo-config

```diff
@@ -1,3 +1,3 @@
 [target.arm-unknown-linux-gnueabihf]
-ar = "arm-linux-gnueabihf-gcc-ar"
+ar = "arm-linux-gnueabihf-ar"
```

Comment:
I had some problems executing the linker and compared it with your solution here:
https://github.com/Ogeon/rust-wiringpi/blob/master/scripts/config

but either work